site/src/pages/UserSettingsPage/WorkspaceProxyPage/WorkspaceProxyRow.tsx

```diff
@@ -109,7 +134,7 @@ const ProxyMessagesRow: FC<ProxyMessagesRowProps> = ({ proxy }) => {
 				title={
 					<span css={{ color: theme.palette.warning.light }}>Warnings</span>
 				}
-				messages={proxy.status?.report?.warnings}
+				messages={proxy.status?.report?.warnings.concat(extraWarnings)}
```

Comment:
```suggestion
				messages={[...(proxy.status?.report?.warnings ?? []), ...extraWarnings]}
```

as is, `extraWarnings` will never be shown if either of `proxy.status` or `proxy.status.report` is `undefined`

Reply:
:+1: